2017-01-09 22:36:31.453997

command:10
loops:python ../userquery/classifybyuser.py ~/data/topics/ ../tfidf/result-new/b232-1-2.txt ~/data/lsaclass/ 0.11 2 0 ../userquery/result/
output-floder:log/

Traceback (most recent call last):
  File "loop.py", line 23, in <module>
    loops = int(sys.argv[2])
ValueError: invalid literal for int() with base 10: 'python ../userquery/classifybyuser.py ~/data/topics/ ../tfidf/result-new/b232-1-2.txt ~/data/lsaclass/ 0.11 2 0 ../userquery/result/'
